tests/verilog-conversion/main_sc_hdl.txt

```diff
@@ -0,0 +1,343 @@
+              (hLiteral coef3)
+            ]
+          ]
+          [
+            [
+              (hVardecl result2)
+              [
+                (hType int)
+              ]
+              [
+                (hBinop +)
+                (hLiteral prod1)
+                (hLiteral prod2)
+              ]
+            ]
+          ]
+          [
+            [
+              (hVardecl result3)
+              [
+                (hType int)
+              ]
+              [
+                (hBinop +)
+                [
+                  (hBinop +)
+                  (hLiteral prod1)
+                  (hLiteral prod2)
+                ]
+                (hLiteral prod3)
+              ]
+            ]
+          ]
+          [
+            (hSigAssignL write)
+            (hLiteral sum2)
+            (hVardecl result2)
+          ]
+          [
+            (hSigAssignL write)
+            (hLiteral sum3)
+            (hVardecl result3)
+          ]
+          [
+            (hBinop =)
+            (hLiteral o_data)
+            [
+              (hSigAssignR read)
+              (hLiteral sum3)
+            ]
+          ]
+        ]
+      ]
+    ]
+  ]
+  [
+    (hModule driver_0)
+    [
+      (hPortout filter_clk)
+      [
+        (hType sc_out)
+        (hType _Bool)
+      ]
+      (hPortout filter_clk1)
+      [
+        (hType sc_out)
+        (hType _Bool)
+      ]
+      (hPortout filter_clk2)
+      [
+        (hType sc_out)
+        (hType _Bool)
+      ]
+      (hPortout filter_clk3)
+      [
+        (hType sc_out)
+        (hType _Bool)
+      ]
+      (hPortout filter_input)
+      [
+        (hType sc_out)
+        (hType sc_bv)
+        (hType 8)
+      ]
+    ]
+    [
+      [
+        (hProcess filter_clk_driver)
+        [
+          (hCStmt )
+          [
+            (hBinop =)
+            (hLiteral filter_clk1)
+            [
+              (hBinop ^)
+              [
+                (hLiteral operator const bool &)
+                (hLiteral filter_clk2)
+              ]
+              [
+                (hLiteral operator const bool &)
+                (hLiteral filter_clk3)
+              ]
+            ]
+          ]
+          [
+            (hBinop =)
+            (hLiteral filter_clk)
+            [
+              (hUnop !)
+              [
+                (hLiteral operator const bool &)
+                (hLiteral filter_clk1)
+              ]
+            ]
+          ]
+          [
+            (hLiteral next_trigger)
+            (hLiteral 4)
+            (hVardecl SC_NS)
+          ]
+        ]
+      ]
+    ]
+  ]
+  [
+    (hModule monitor_0)
+    [
+      (hPortin filter_input)
+      [
+        (hType sc_in)
+        (hType sc_bv)
+        (hType 8)
+      ]
+      (hPortin filter_output)
+      [
+        (hType sc_in)
+        (hType sc_bv)
+        (hType 8)
+      ]
+    ]
+    [
+      [
+        (hProcess prc_monitor)
+        [
+          (hCStmt )
+        ]
+      ]
+    ]
+  ]

```

Comment:
The checked in version of this file has the sensitivity lists. Also for that operator const bool & it uses the opcode hNoop. Did you want to check in this version and then re-generate when you are caught up with master branch?

Reply:
In the latest commit, such files should be removed because they were under `tests/verilog-conversion` which only includes python scripts (no generated file at this point).

So yes, I would like to re-generate these files in future commit.

Thanks